pkg/config/release.go

```diff
@@ -227,20 +227,25 @@ func GetChangedRegistrySteps(path, baseRev string, graph registry.NodeByName) ([
 }
 
 func GetChangedClusterProfiles(path, baseRev string) ([]string, error) {
-	return getRevChanges(path, ClusterProfilesPath, baseRev)
+	return getRevChanges(path, ClusterProfilesPath, baseRev, false)
 }
 
-func GetChangedConfigs(path, baseRev string) ([]string, error) {
-	return getRevChanges(path, CiopConfigInRepoPath, baseRev)
+func GetAddedConfigs(path, baseRev string) ([]string, error) {
+	return getRevChanges(path, CiopConfigInRepoPath, baseRev, true)
 }
 
 // getRevChanges returns the name and a hash of the contents of files under
 // `path` that were added/modified since revision `base` in the repository at
 // `root`.  Paths are relative to `root`.
-func getRevChanges(root, path, base string) ([]string, error) {
+// If 'ignoreModified' is true it will only check for relevant added, moved, or copied files
+func getRevChanges(root, path, base string, ignoreModified bool) ([]string, error) {
 	// Sample output (with abbreviated hashes) from git-diff-tree(1):
 	// :100644 100644 bcd1234 0123456 M file0
-	cmd := []string{"diff-tree", "-r", "--diff-filter=d", base + ":" + path, "HEAD:" + path}
+	filter := "--diff-filter=d"
+	if ignoreModified {
+		filter = "--diff-filter=ACR"
```

Comment:
Why removed `d`?

Reply:
the lowercase `d` means everything **but** deleted. using uppercase letters means we **only** want the options supplied. In this case: added, moved, or copied.